gui/quantum.lua

```diff
@@ -11,262 +11,218 @@ local quickfort = reqscript('quickfort')
 local quickfort_command = reqscript('internal/quickfort/command')
 local quickfort_orders = reqscript('internal/quickfort/orders')
 
-QuantumUI = defclass(QuantumUI, guidm.MenuOverlay)
+
+-- UI Layout
+QuantumUI = defclass(QuantumUI, gui.ZScreen)
 QuantumUI.ATTRS {
-    frame_inset=1,
     focus_path='quantum',
-    sidebar_mode=df.ui_sidebar_mode.LookAround,
+    pass_movement_keys=true,
+    pass_mouse_clicks=false
 }
 
-function QuantumUI:init()
-    local cart_count = #assign_minecarts.get_free_vehicles()
+QuantumWin = defclass(QuantumWin, widgets.Window)
+QuantumWin.ATTRS {
+    frame_title = 'Quantum Stockpile',
+    frame = { w=60, h=26 },
+    resizable = true,
+    autoarrange_subviews=true
+}
 
-    local main_panel = widgets.Panel{autoarrange_subviews=true,
-                                     autoarrange_gap=1}
-    main_panel:addviews{
-        widgets.Label{text='Quantum'},
+function QuantumWin:init()
+    cart_count = #assign_minecarts.get_free_vehicles()
```

Comment:
`cart_count` should be an instance variable, not a global. call it `self.cart_count` instead